gmd-sdk/nifi-data-processors/pom.xml

```diff
@@ -77,12 +77,6 @@
           <artifactId>refined-scalaz_${scala.major.version}</artifactId>
           <version>${refined-scalaz.version}</version>
         </dependency>
-        <dependency>
```

Comment:
Not sure why this is removed as part of this pull request.  Doesn't seem related to the issues being resolved.

Reply:
It was just unnecessary as it was added to the parent pom long ago and I am back to modifying that file. But you are correct that it doesn't really fit with the issues being fixed.